recipes/jaws/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "jaws" %}
+{% set version = "0.2.3" %}
+{% set sha256 = "b8cd6e76f01a3786d32880c757b02aa82651eb9fffeb0bc8aa155dac02212b38" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jaws/jaws/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pip
+    - numpy >=1.11
+  run:
+    - python
+    - pandas
+    - numpy
+    - xarray
+    - pytz
+    - netcdf4
+    - matplotlib
+    - netcdf4
+
+test:
+  source_files:
+    - sample_data/GCNET_Summit_20140601.txt
+  imports:
+    - source
```

Comment:
I think this needs to be `jaws` since that is the name of the package. These imports are from a python session in a directory without the source code, so the source/tests that are being pulled in are not from `jaws` source code.